src/tainting/Dataflow_tainting.mli

```diff
@@ -98,14 +103,19 @@ val hook_function_taint_signature :
   (AST_generic.parameters (* params of function *) * Taint.finding list) option)
   option
   ref
-(** Deep Semgrep *)
+(** DEEP *)
+
+val hook_find_attribute_in_class :
+  (AST_generic.type_ -> string -> AST_generic.name option) option ref
```

Comment:
this is an aside, but we should start aliasing `option ref` for hooks, to like `'a hook` or something. it's more clear.

Reply:
I would like to get rid of the "hooks" and just pass functions via some config object. Not a fan of this TBH.